### PR TITLE
Add support for titles as described by the metadata department

### LINKED ISSUE #1034: Map MODS alternative title to cocina descriptive
See https://github.com/sul-dlss-labs/cocina-descriptive-metadata/blob/master/mods_cocina_mappings/mods_to_cocina_titleInfo.txt#L58

```xml
  <titleInfo type="alternative">
    <title>Floreal</title>
  </titleInfo>
```

### LINKED ISSUE #1035: Map MODS title with parts to cocina
See https://github.com/sul-dlss-labs/cocina-descriptive-metadata/blob/master/mods_cocina_mappings/mods_to_cocina_titleInfo.txt#L16

```xml
  <titleInfo>
    <title>Compose</title>
    <subTitle>for two pianos and orchestral groups. Floreal, for orchestra. Changeant, for cello and orchestra. Hommage a Heinrich Schuetz. Surprise for string orchestra</subTitle>
  </titleInfo>

```

### LINKED ISSUE #1036: Map basic title to cocina
https://github.com/sul-dlss-labs/cocina-descriptive-metadata/blob/master/mods_cocina_mappings/mods_to_cocina_titleInfo.txt#L3-L14

### LINKED ISSUE #1039: Map MODS translated title to cocina
https://github.com/sul-dlss-labs/cocina-descriptive-metadata/blob/master/mods_cocina_mappings/mods_to_cocina_titleInfo.txt#L79-L154

### LINKED ISSUE #1040: Map MODS transliterated title to Cocina
https://github.com/sul-dlss-labs/cocina-descriptive-metadata/blob/master/mods_cocina_mappings/mods_to_cocina_titleInfo.txt#L156-L210

### LINKED ISSUE #1041: Map mods parallel translation to Cocina
https://github.com/sul-dlss-labs/cocina-descriptive-metadata/blob/master/mods_cocina_mappings/mods_to_cocina_titleInfo.txt#L275-L328

### LINKED ISSUE #1042: Map multiple untyped titles without primary to cocina
https://github.com/sul-dlss-labs/cocina-descriptive-metadata/blob/master/mods_cocina_mappings/mods_to_cocina_titleInfo.txt#L330-L346

### LINKED ISSUE #1043: Support mapping multiple typed titles without primary to Cocina
https://github.com/sul-dlss-labs/cocina-descriptive-metadata/blob/master/mods_cocina_mappings/mods_to_cocina_titleInfo.txt#L348-L365

### LINKED ISSUE #1044: Support mapping titles with displayLabel to cocina
https://github.com/sul-dlss-labs/cocina-descriptive-metadata/blob/master/mods_cocina_mappings/mods_to_cocina_titleInfo.txt#L367-L386

### LINKED ISSUE #1045: Support mapping abbreviated titles to Cocina
https://github.com/sul-dlss-labs/cocina-descriptive-metadata/blob/master/mods_cocina_mappings/mods_to_cocina_titleInfo.txt#L251-L273

### LINKED ISSUE #1046: Support mapping supplied titles to Cocina
https://github.com/sul-dlss-labs/cocina-descriptive-metadata/blob/master/mods_cocina_mappings/mods_to_cocina_titleInfo.txt#L237-L249

### LINKED ISSUE #1047: Support mapping uniform title with authority to Cocina
https://github.com/sul-dlss-labs/cocina-descriptive-metadata/blob/master/mods_cocina_mappings/mods_to_cocina_titleInfo.txt#L212-L235

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made?

https://github.com/sul-dlss-labs/cocina-descriptive-metadata/blob/master/mods_cocina_mappings/mods_to_cocina_titleInfo.txt#L3-L14

1. Fixes #1034
2. Fixes #1035
3. Fixes #1036 
4. Fixes #1039
5. Fixes #1040 
6. Fixes #1047
7. Fixes #1046
8. Fixes #1045
9. Fixes #1041
10. Fixes #1042
11. Fixes #1043 
12. Fixes #1044

## How was this change tested?

Test suite

### CODE REVIEW

File: app/services/cocina/from_fedora/descriptive/contributor.rb [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
```
Comment (jcoyne) [inline app/services/cocina/from_fedora/descriptive/contributor.rb:1]: This code is just moved from ContributorMapper, but no functional changes in here.

File: app/services/cocina/from_fedora/descriptive/form.rb [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module Cocina
+  module FromFedora
+    class Descriptive
+      # Maps relevant mods:physicalDescription from descMetadata to cocina
+      class Form
```
Comment (jcoyne) [inline app/services/cocina/from_fedora/descriptive/form.rb:7]: This code is just moved from ContributorMapper, but no functional changes in here

File: app/services/cocina/from_fedora/descriptive/language.rb [hunk 3]
```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module Cocina
+  module FromFedora
+    class Descriptive
+      # Maps languages
+      class Language
```
Comment (jcoyne) [inline app/services/cocina/from_fedora/descriptive/language.rb:7]: This code is just moved from LanguageMapper, but no functional changes in here

File: app/services/cocina/from_fedora/descriptive/notes.rb [hunk 4]
```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module Cocina
+  module FromFedora
+    class Descriptive
+      # Maps notes
+      class Notes
```
Comment (jcoyne) [inline app/services/cocina/from_fedora/descriptive/notes.rb:7]: This code is just moved from NotesMapper, but no functional changes in here

**Review by mjgiarlo — APPROVED** [review]
LGTM. Please announce in change mgmt slack channel, too?

**Pull request merged**